modules/virt-creating-a-service-from-a-virtual-machine.adoc

```diff
@@ -8,6 +8,26 @@
 
 Create a service from a running virtual machine by first creating a `Service` object to expose the virtual machine.
 
+[NOTE]
+====
+If IPv4 and IPv6 dual-stack networking is enabled for your cluster, you can create a service that can use IPv4, IPv6, or both, by defining the `spec.ipFamilyPolicy` and the `spec.ipFamilies` fields in the `Service` object.
+
+The `spec.ipFamiliPolicy` field can be set to one of the following values:
+
+* `SingleStack`: The control plane assigns a cluster IP address for the service based on the first configured service cluster IP range.
+
+* `PreferDualStack`: The control plane assigns both IPv4 and IPv6 cluster IP addresses for the service on clusters that have dual-stack configured.
+
+* `RequireDualStack`: This option fails for clusters that do not have dual-stack networking enabled. For clusters that have dual-stack configured, the behavior is the same as when the value is set to `PreferDualStack`. The control plane allocates cluster IP addresses from both IPv4 and IPv6 address ranges.
+
+You can define which IP family to use for single-stack or define the order of IP families for dual-stack, by setting the `spec.ipFamilies` field to one of the following array values:
```

Comment:
`You can define which IP family to use for single-stack or define the order of IP families for dual-stack,`

I think you should either remove the comma or add a comma after `single-stack`. My preference would be for removing the comma.